server/src/main/scala/com/cloudera/livy/utils/LivySparkUtils.scala

```diff
@@ -100,7 +100,7 @@ object LivySparkUtils extends Logging {
       pb.environment().put("LIVY_TEST_CLASSPATH", sys.props("java.class.path"))
     }
 
-    val process = new LineBufferedProcess(pb.start())
+    val process = new LineBufferedProcess(pb.start(), 200)
```

Comment:
I think it runs only when livy server is started. Do we need to make it config driven?
If the number is set to too low by mistake, the livy server mayn't start.